packit_service/worker/parser.py

```diff
@@ -378,7 +453,6 @@ def parse_release_event(event) -> Optional[ReleaseEvent]:
         """
         https://developer.github.com/v3/activity/events/types/#releaseevent
         https://developer.github.com/v3/repos/releases/#get-a-single-release
-
```

Comment:
I would leave those free spaces here for better readability.

Reply:
Still relevant...

Reply:
okay.